extensions/PhabBugz/lib/Util.pm

```diff
@@ -0,0 +1,211 @@
+            },
+#            {
+#                action => 'allow',
+#                rule   => 'PhabricatorUsersPolicyRule',
+#                value  => $user_phids,
+#            }
+        ]
+    };
+
+    my $result = request('policy.create', $data);
+    return $result->{result}{phid};
+}
+
+sub make_revision_public {
+    my ($revision_phid) = @_;
+    return request('differential.revision.edit', {
+        transactions => [
+            {
+                type  => "view",
+                value => "users"
+            }
+        ],
+        objectIdentifier => $revision_phid
+    });
+}
+
+sub edit_revision_policy {
+    my ($revision_phid, $policy_phid) = @_;
+
+    my $data = {
+        transactions => [
+            {
+                type  => 'view',
+                value => $policy_phid
+            }
+        ],
+        objectIdentifier => $revision_phid
+    };
+
+    return request('differential.revision.edit', $data);
+}
+
+sub get_project_phid {
+    my $project = shift;
+
+    my $data = {
+        queryKey => 'active',
+        constraints => {
+            name => $project
+        }
+    };
+
+    my $result = request('project.search', $data);
+    if (!$result->{result}{data}) {
+        return undef;
+    }
+    return $result->{result}{data}[0]{phid};
+}
+
+sub create_project {
+    my ($project, $description, $members) = @_;
+
+    my $data = {
+        transactions => [
+            { type => 'name',  value => $project           },
+            { type => 'description', value => $description },
+            { type => 'edit',  value => 'admin'            },
+            { type => 'join',  value => 'admin'            },
+            { type => 'view',  value => 'admin'            },
+            { type => 'icon',  value => 'group'            },
+            { type => 'color', value => 'red'              }
+        ]
+    };
+
+    my $result = request('project.edit', $data);
+    return $result->{result}{object}{phid};
+}
+
+sub set_project_members {
+    my ($project_id, $phab_user_ids) = @_;
+
+    my $data = {
+        objectIdentifier => $project_id,
+        transactions => [
+            { type => 'members.set',  value => $phab_user_ids }
+        ]
+    };
+
+    my $result = request('project.edit', $data);
+    return $result->{result}{object}{phid};
+}
+
+sub get_members_by_bmo_id {
+    my $users = shift;
+
+    my $data = {
+        accountids => [ map { $_->id } @$users ]
+    };
+
+    my $result = request('bmoexternalaccount.search', $data);
+    if (!$result->{result}) {
+        return [];
+    }
+
+    my @phab_ids;
+    foreach my $user (@{ $result->{result} }) {
+        push(@phab_ids, $user->{phid});
+    }
+    return \@phab_ids;
+}
+
+sub request {
+    my ($method, $data) = @_;
+
+    my $phab_api_key  = Bugzilla->params->{phabricator_api_key};
+    my $phab_base_uri = Bugzilla->params->{phabricator_base_uri};
+    $phab_base_uri || ThrowUserError('invalid_phabricator_uri');
```

Comment:
I prefer:
```perl

ThrowUserError('invalid_phabricator_uri') unless $phab_base_uri;
```

As the main point of the code is throwing an error, not the variable name.